### PR TITLE
[docs] Update uuid-ossp extension

### PR DESCRIPTION
**Pull request by ddhodge**
Update uuid-ossp extension info; various formatting and links in Explore/SQL features sections.
Backport to
v2.12/api/ysql/extensions/#uuid-ossp
stable/explore/ysql-language-features/pg-extensions/

@netlify /preview/explore/ysql-language-features/pg-extensions

### CODE REVIEW

File: docs/content/preview/explore/ysql-language-features/triggers.md [hunk 1]
```diff
@@ -213,7 +213,7 @@ ALTER TABLE employees
   ENABLE TRIGGER ALL;
 ```
 
-## Using Event Triggers
+## Event triggers
```
Comment (aishwarya24) [inline docs/content/preview/explore/ysql-language-features/triggers.md:216]: change Creating Triggers link on line 162 and 230 to (#create-triggers)

File: docs/content/v2.12/api/ysql/extensions.md [hunk 2]
```diff
@@ -241,6 +241,27 @@ The `crosstab()`and  `crosstabN()` functions produce “pivot” displays. The _
...
 For more information, refer to [`tablefunc`](https://www.postgresql.org/docs/11/tablefunc.html) in the PostgreSQL docs.
 
+### uuid-ossp
+
+The [`uuid-ossp`](https://www.postgresql.org/docs/current/uuid-ossp.html) extension provides functions to generate universally unique identifiers (UUIDs), and functions to produce certain special UUID constants.
+
+#### uuid-ossp example
+
+```sql
+CREATE EXTENSION uuid-ossp;
```
Comment (aishwarya24) [inline docs/content/v2.12/api/ysql/extensions.md:251]:
```suggestion
CREATE EXTENSION "uuid-ossp";
```

Comment (aishwarya24) [reply docs/content/v2.12/api/ysql/extensions.md:251]: on preview I got the syntax error. 2.12 may have it as well.

**Review by aishwarya24 — COMMENTED** [review]
Left some comments.

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Review by sanyamsinghal — APPROVED** [review]
LGTM (I mainly looked at pg-extensions.md file changes)

**Pull request merged**